### ISSUE TITLE
[question] Having trouble running specs

### ISSUE THREAD

**Issue by npafundi** [liveeditor/draftsman#13]
Hey Chris,

I forked draftsman to add a small piece of functionality, and wanted to send you a pull request.  However, I'm having trouble getting the specs to run, and didn't want to send a pull request without running (and adding) tests.

When I run `RAILS_ENV=test rake db:migrate` (or other db and spec tasks), I get:

```
rake aborted!
LoadError: cannot load such file -- rack/test
/<my path>/draftsman/spec/dummy/config/application.rb:4:in `<top (required)>'
/<my path>/draftsman/spec/dummy/Rakefile:5:in `<top (required)>'
LoadError: cannot load such file -- active_record/railtie
/<my path>/draftsman/spec/dummy/config/application.rb:4:in `<top (required)>'
/<my path>/draftsman/spec/dummy/Rakefile:5:in `<top (required)>'
```

I was following your listed steps in the "Contributing" section of the docs.  Sorry if I'm missing something obvious here!

Thanks again!
Nick

**Comment by chrisdpeters** [comment]
I believe this is because of the last part of my note in issue #8. I need to fix this ASAP. I will look at it this week.

Sorry about that.

**Comment by npafundi** [comment]
Great, thanks for letting me know!

**Comment by chrisdpeters** [comment]
Hmm, now that I look at your error closer, I think it's something else. Can you comment with your `Gemfile.lock` so I can see what versions of the different dependencies you're running?

**Comment by npafundi** [comment]
If I run a `bundle install`, it changes only two lines in the `Gemfile.lock`.  `draftsman` is bumped to `0.3.1` (from 0.3.0), and `arel` is bumped to 4.0.2, from `4.0.1`.

Here's the whole file -- I added comments on the two changed lines:

```PATH
  remote: .
  specs:
    draftsman (0.3.1)                       # was `draftsman (0.3.0)`
      activerecord (>= 3.0, < 5.0)

GEM
  remote: https://rubygems.org/
  specs:
    actionpack (4.0.1)
      activesupport (= 4.0.1)
      builder (~> 3.1.0)
      erubis (~> 2.7.0)
      rack (~> 1.5.2)
      rack-test (~> 0.6.2)
    activemodel (4.0.1)
      activesupport (= 4.0.1)
      builder (~> 3.1.0)
    activerecord (4.0.1)
      activemodel (= 4.0.1)
      activerecord-deprecated_finders (~> 1.0.2)
      activesupport (= 4.0.1)
      arel (~> 4.0.0)
    activerecord-deprecated_finders (1.0.3)
    activesupport (4.0.1)
      i18n (~> 0.6, >= 0.6.4)
      minitest (~> 4.2)
      multi_json (~> 1.3)
      thread_safe (~> 0.1)
      tzinfo (~> 0.3.37)
    arel (4.0.2)                         # was `arel (4.0.1)`
    atomic (1.1.14)
    builder (3.1.4)
    capybara (2.1.0)
      mime-types (>= 1.16)
      nokogiri (>= 1.3.3)
      rack (>= 1.0.0)
      rack-test (>= 0.5.4)
      xpath (~> 2.0)
    diff-lcs (1.2.5)
    erubis (2.7.0)
    i18n (0.6.5)
    mime-types (2.0)
    mini_portile (0.5.2)
    minitest (4.7.5)
    multi_json (1.8.2)
    nokogiri (1.6.0)
      mini_portile (~> 0.5.0)
    rack (1.5.2)
    rack-protection (1.5.1)
      rack
    rack-test (0.6.2)
      rack (>= 1.0)
    railties (4.0.1)
      actionpack (= 4.0.1)
      activesupport (= 4.0.1)
      rake (>= 0.8.7)
      thor (>= 0.18.1, < 2.0)
    rake (10.1.0)
    rspec-core (2.14.7)
    rspec-expectations (2.14.4)
      diff-lcs (>= 1.1.3, < 2.0)
    rspec-mocks (2.14.4)
    rspec-rails (2.14.0)
      actionpack (>= 3.0)
      activesupport (>= 3.0)
      railties (>= 3.0)
      rspec-core (~> 2.14.0)
      rspec-expectations (~> 2.14.0)
      rspec-mocks (~> 2.14.0)
    shoulda-matchers (2.4.0)
      activesupport (>= 3.0.0)
    sinatra (1.4.4)
      rack (~> 1.4)
      rack-protection (~> 1.4)
      tilt (~> 1.3, >= 1.3.4)
    sqlite3 (1.3.8)
    thor (0.18.1)
    thread_safe (0.1.3)
      atomic
    tilt (1.4.1)
    tzinfo (0.3.38)
    xpath (2.0.0)
      nokogiri (~> 1.3)

PLATFORMS
  ruby

DEPENDENCIES
  capybara
  draftsman!
  railties (>= 3.0, < 5.0)
  rake
  rspec-rails
  shoulda-matchers
  sinatra (~> 1.0)
  sqlite3 (~> 1.2)```

**Comment by npafundi** [comment]
Hey @chrisdpeters, I pulled in all the recent fixes and upgraded to ruby 2.2.1, then tried running the specs again.  That ended up fixing the issues above, but now the specs can't load draftsman:

```
LoadError: cannot load such file -- draftsman
<my path>/draftsman/spec/dummy/config/application.rb:9:in `<top (required)>'
<my path>/draftsman/spec/dummy/Rakefile:5:in `<top (required)>'
```

I ran a `gem install draftsman`, and then I was finally able to create the database.  Moved back to the draftsman root, and it looks like I can finally run the tests.

Is there something else I should be able to do, rather than manually install the gem?
